### PR TITLE
rasterize-image to return pdf

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25711

## Description
resterize-image returned png instead of pdf. not it returns PDF.

Failed circle:
https://app.circleci.com/pipelines/github/demisto/content/16732/workflows/3ae45262-4ec6-4d0e-bd31-80d1e7a4a38b/jobs/59126/steps

## Screenshots
image

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 1]
```diff
@@ -306,17 +306,18 @@ def rasterize_command():
...
-    w = demisto.args().get('width', DEFAULT_W).rstrip('px')
-    h = demisto.args().get('height', DEFAULT_H).rstrip('px')
+    args = demisto.args()
+    entry_id = args.get('EntryID')
+    w = args.get('width', DEFAULT_W).rstrip('px')
+    h = args.get('height', DEFAULT_H).rstrip('px')
 
     file_path = demisto.getFilePath(entry_id).get('path')
-    filename = 'image.png'  # type: ignore
+    filename = 'document.pdf'
```
Comment (Itay4) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:315]:
```suggestion
    filename = 'entry_id.pdf'
```

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 2]
```diff
@@ -306,17 +306,18 @@ def rasterize_command():
...
-    w = demisto.args().get('width', DEFAULT_W).rstrip('px')
-    h = demisto.args().get('height', DEFAULT_H).rstrip('px')
+    args = demisto.args()
+    entry_id = args.get('EntryID')
+    w = args.get('width', DEFAULT_W).rstrip('px')
+    h = args.get('height', DEFAULT_H).rstrip('px')
 
     file_path = demisto.getFilePath(entry_id).get('path')
-    filename = 'image.png'  # type: ignore
+    filename = 'entry_id.pdf'
```
Comment (Itay4) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:315]:
```suggestion
    filename = f'{entry_id}.pdf'
```

**Comment by Itay4** [comment]
is this ready for merge? why is this marked "in progress"?

**Comment by jochman** [comment]
fails on maximum mailbox size:
image

@idovandijk FYI

**Comment by Itay4** [comment]
@roysagi please review rn

**Comment by Itay4** [comment]
build fails on marketplace
merging

**Pull request merged**